tests/unit/createRouter.ts

```diff
@@ -149,13 +149,14 @@ suite('createRouter', () => {
 	test('dispatch emits navstart event', () => {
 		const router = createRouter();
 
-		let received = '';
+		let received: NavigationStartEvent = null!;
```

Comment:
I'd like to avoid `any`, it can start to hide typing errors in API usage.

I'm hoping we get used to the exclamation mark. It'd require a lot of comments otherwise…